dictionaries/CallMap_80_delta.php

```diff
@@ -97,6 +97,10 @@
       'old' => ['array<string,mixed>'],
       'new' => ['array<string,mixed>', 'filter='=>'?int'],
     ],
+    'ReflectionClass::getReflectionConstants' => [
+        'old' => ['array<string,mixed>'],
+        'new' => ['array<string,mixed>', 'filter='=>'?int'],
```

Comment:
Seems like there is an incoherence between the signature here `array<string,mixed>` and int CallMap.php `list<ReflectionClassConstant>`